docs/CONTRIBUTING.md

```diff
@@ -110,35 +122,35 @@ $ make test
 
 #### Run e2e tests
 
-First you should build a local Kubernetes environment for e2e tests.
-
-* Use kind to build Kubernetes cluster
-
-    Please ensure you have install [kind](https://kind.sigs.k8s.io/) and it's version == v0.4.0
+You can run the following command to run all e2e test.
 
-    Use the following command to create a local kind Kubernetes environment.
-
-    ```sh
-    $ ./hack/kind-cluster-build.sh
-    ```
+```sh
+$ make e2e
+```
 
-    If you have customization requirements, please refer the help info:
+It's possible to limit specs to run, for example:
 
-    ```
-    $ ./hack/kind-cluster-build.sh --help
-    ```
+```
+./hack/e2e.sh -- --ginkgo.focus='Basic'
```

Comment:
`e2e.sh` cannot run out of box on Darwin, we should document this

Reply:
is it hard to support Darwin?

Reply:
No, it is not hard, the reason is the some arguments of GNU command tools do no work on BSD suite

Reply:
I did a quick test on Darwin, it can run now. I guess this PR fix the issue:https://github.com/pingcap/tidb-operator/pull/1285/files

Reply:
can you verify it on your machine?

Reply:
BSD `xargs` do not support  `-n` used here https://github.com/pingcap/tidb-operator/blob/2dd0f545f35c69bda74f92f8576ea875727e9fff/hack/run-e2e.sh#L63
```
xargs -n
xargs: option requires an argument -- n
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements]] [-J replstr]
             [-L number] [-n number [-x]] [-P maxprocs] [-s size]
             [utility [argument ...]]
```

Reply:
I'm going to move these code to e2e code. Better to keep the shell script simple.

Reply:
fixed in 4c97b4a4530b4c95df7b25a88290c5d5b855320a.